### PR TITLE
move makejdk.sh to sbin, update manpage

### PR DESCRIPTION
**Pull request by gdams**
closes: https://github.com/AdoptOpenJDK/openjdk-build/issues/120 #121

### CODE REVIEW

**Comment by gdams** [comment]
https://ci.adoptopenjdk.net/view/OpenJDK%20PR/job/openjdk_pr_pipeline/13/

**Comment by gdams** [comment]
@a-roberts updated PTAL

File: makejdk-any-platform.1 [hunk 1]
```diff
@@ -1,11 +1,14 @@
-.\" Manpage for makejdk.sh
+.\" Manpage for makejdk-any-platform.sh
```
Comment (neomatrix369) [inline makejdk-any-platform.1]: :+1:

File: sbin/makejdk.sh [hunk 2]
```diff
@@ -53,7 +53,7 @@ determineBuildProperties
 sourceFileWithColourCodes()
 {
   # shellcheck disable=SC1091
-  source ./sbin/colour-codes.sh
+  source sbin/colour-codes.sh
```
Comment (neomatrix369) [inline sbin/makejdk.sh]: this might not work - we are now in the `$SCRIPT_DIR`, arent we?

File: sbin/makejdk.sh [hunk 3]
```diff
@@ -331,7 +331,7 @@ testOpenJDKInNativeEnvironmentIfExpected()
 buildAndTestOpenJDKInNativeEnvironment()
 {
   echo "Calling sbin/build.sh $WORKING_DIR $TARGET_DIR $OPENJDK_REPO_NAME $JVM_VARIANT $OPENJDK_UPDATE_VERSION $OPENJDK_BUILD_NUMBER"
-  "${SCRIPT_DIR}"/sbin/build.sh "${WORKING_DIR}" "${TARGET_DIR}" "${OPENJDK_REPO_NAME}" "${JVM_VARIANT}" "${OPENJDK_UPDATE_VERSION}" "${OPENJDK_BUILD_NUMBER}"
+  "${SCRIPT_DIR}"/build.sh "${WORKING_DIR}" "${TARGET_DIR}" "${OPENJDK_REPO_NAME}" "${JVM_VARIANT}" "${OPENJDK_UPDATE_VERSION}" "${OPENJDK_BUILD_NUMBER}"
```
Comment (neomatrix369) [inline sbin/makejdk.sh]: /build.sh ==> `./build.sh` or just `build.sh`

File: makejdk-any-platform.1 [hunk 4]
```diff
@@ -1,11 +1,14 @@
...
 .TH "0.1.0" "Apache 2"
 .SH NAME
-makejdk.sh
+makejdk-any-platform.sh
 .SH SYNOPSIS
-"./makejdk [options]"
+"makejdk-any-platform.sh [options]"
 .SH DESCRIPTION
-The simplest way to build OpenJDK using our scripts is to run makejdk.sh and have your user be in the Docker group on the machine (or prefix all of your Docker commands with sudo. This script can be used to create a Docker container that will be configured with all of the required dependencies and a base operating system in order to build OpenJDK.
+The simplest way to build OpenJDK using our scripts is to run makejdk-any-platform.sh and have your user be in the Docker group on the machine (or prefix all of your Docker commands with sudo. This script can be used to create a Docker container that will be configured with all of the required dependencies and a base operating system in order to build OpenJDK.
```
Comment (neomatrix369) [inline makejdk-any-platform.1]: have your user be in the ==> have your $USER be added to the Docker group....

Comment (neomatrix369) [inline makejdk-any-platform.1]: Script can be used to create a Docker image ==> its a container when we run a Docker image (nit picking)

File: makejdk-any-platform.1 [hunk 5]
```diff
@@ -1,11 +1,14 @@
...
+.SH ENVIRONMENT VARIABLES
+.BR CONFIGURE_ARGS_FOR_ANY_PLATFORM
```
Comment (neomatrix369) [inline makejdk-any-platform.1]: Add an example usage like @a-roberts suggested:
```
CONFIGURE_ARGS_FOR_ANY_PLATFORM e.g.
--with-extra-ldflags=<foo> --with-extra-cflags=<bar> --with-extra-cxxflags=<gwas>
```

File: sbin/makejdk.sh (hunk 2 above)
Comment (gdams) [inline sbin/makejdk.sh]: this seemed to work for me, if you look at the test runs the colours appear

File: sbin/makejdk.sh (hunk 3 above)
Comment (gdams) [inline sbin/makejdk.sh]: what's wrong with the current syntax here? just passing it the full path

**Comment by gdams** [comment]
@neomatrix369 updated PTAL

File: sbin/makejdk.sh (hunk 2 above)
Comment (neomatrix369) [inline sbin/makejdk.sh]: Safer to add the `$SCRIPT_DIR` i think

File: sbin/makejdk.sh (hunk 3 above)
Comment (neomatrix369) [inline sbin/makejdk.sh]: Ignore me I was looking at the wrong thing, you are right its good the way it is

**Comment by neomatrix369** [comment]
@gdams I think this also closes #121, no?

File: sbin/makejdk.sh (hunk 2 above)
Comment (gdams) [inline sbin/makejdk.sh]: updated PTAL

**Comment by gdams** [comment]
CI: https://ci.adoptopenjdk.net/view/OpenJDK PR/job/openjdk_pr_pipeline/18

**Comment by gdams** [comment]
hmmmm @neomatrix369 linter doesn't like those last changes
``` bash
In sbin/makejdk.sh line 56:

  source "${SCRIPT_DIR}"/colour-codes.sh

  ^-- SC1090: Can't follow non-constant source. Use a directive to specify location.

In sbin/makejdk.sh line 61:

  source "${SCRIPT_DIR}"/signalhandler.sh

  ^-- SC1090: Can't follow non-constant source. Use a directive to specify location.

```

**Comment by tellison** [comment]
You should add a directive to show shellcheck where the files can be found, e.g. as they are now in the same dir I guess you can simply write,
```
# shellcheck source=colour-codes.sh
source "${SCRIPT_DIR}"/colour-codes.sh
```

**Comment by neomatrix369** [comment]
@gdams https://travis-ci.org/AdoptOpenJDK/openjdk-build/builds/243639679?utm_source=github_status&utm_medium=notification - see if you can resolve these :) thnx

**Comment by karianna** [comment]
@gdams this needs a rebase and push

**Comment by gdams** [comment]
updated PTAL

**Comment by karianna** [comment]
@gdams Still merge conflicts?

**Pull request closed without merging**